sdk/metric/controller/basic/controller_test.go

```diff
@@ -83,6 +83,10 @@ func TestControllerUsesResource(t *testing.T) {
 			},
 			wanted: "R=S",
 		},
+		{
+			name:    "explicitly environment resource",
+			options: []controller.Option{controller.WithResource(resource.Environment())},
+			wanted:  resource.Environment().Encoded(attribute.DefaultEncoder())},
```

Comment:
Shouldn't this check happen in the `explicitly empty resource` test case above?  It seems like you'd want to add something to the `OTEL_RESOURCE_ATTRIBUTES` environment variable and check that the expected attributes appeared here.

Reply:
Yes, I will add attributes to the OTEL_RESOURCE_ATTRIBUTES environment variable and modify it